src/table/components/TableWrapper.jsx

```diff
@@ -125,7 +126,8 @@ export default function TableWrapper(props) {
         })
       }
     >
-      <div id="sn-table-announcer" aria-live="polite" aria-atomic="true" className={classes.screenReaderOnly} />
```

Comment:
i thinks it would be better if we have a component that holds these two elements 
are you agree?

Reply:
yes that would be nice

Reply:
✅   Fixed